index.html

```diff
@@ -1,12 +1,13 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
- <meta charset="UTF-8">
- <meta http-equiv="X-UA-Compatible" content="IE=edge">
- <meta name="viewport" content="width=device-width, initial-scale=1.0">
- <title>Potionfy</title>
-</head>
-<body>
- 
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Potionfy</title>
+  </head>
+  <body>
+    <h1>Welcome to potionfy</h1>
```

Comment:
Can you remove 'Enjoy your stay string and make the welcome to pontify!? string red?

Reply:
Yeah, sure!